.github/ISSUE_TEMPLATE/default-new.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Issue
+about: This template is for a generic new issue.
+---
```

Comment:
*Suggestion:*

Add an `assignees` here for those that will be automatically assigned. Otherwise we run the danger no one is tagged and issues go stale! Perhaps the co-leads for this project team?

Reply:
Yes, I agree please automatically assign the us

Reply:
Okay, so the assignees would be @Marie-L and @AbigailMesrenyameDogbe ?

Reply:
yes, please add us 😊